superset/views/core.py

```diff
@@ -2498,7 +2503,7 @@ def fetch_datasource_metadata(self) -> FlaskResponse:  # pylint: disable=no-self
     @event_logger.log_this
     @expose("/queries/<float:last_updated_ms>")
     @expose("/queries/<int:last_updated_ms>")
-    @deprecated()
+    @deprecated(new_target="api/v1/query/updated_since")
```

Comment:
nit, but I noticed that some of these new targets have trailing slashes and some don't. is that the way the api is set up, do you know?

Reply:
On FAB `ModelRestApi` CRUD endpoints are created given the following logic:
- All endpoints at the root of a resource name have a trailing slash, example:
  GET `/api/v1/dataset/`
  POST `/api/v1/dataset/`
- All endpoints that contain a path parameter don't have a trailing slash, example:
   GET `/api/v1/dataset/1`
   PUT `/api/v1/dataset/1`

On related nested resource names:
GET `/api/v1/dataset/1/column/`
GET `/api/v1/dataset/1/column/1`

This enforces a base cohesion, yet not a global cohesion. Currently on Superset theres probably some confusion around this. I'll take a pass on the most recent migrations, to check if there's some obvious changes ASAP